tests/workflows/test_rolling_release_workflow.py

```
@@ -237,6 +261,87 @@ def test_publish_job_runs_even_if_build_lints_fails(workflow_text: str) -> None:
+        "force_dependency_binary_rebuild must remain optional for manual "
+        "rolling-release republishes"
+    )
+    assert input_mapping.get("default") is False, (
+        "force_dependency_binary_rebuild must default to false so manual runs "
+        "do not rebuild dependency binaries implicitly"
+    )
+
+    description = input_mapping.get("description")
+    assert isinstance(description, str) and description.strip(), (
+        "force_dependency_binary_rebuild must document its recovery purpose"
+    )
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

(PT018)